src/mission_start.cpp

```diff
@@ -142,6 +142,21 @@ void mission_start::infect_npc( mission *miss )
     p->guard_current_pos();
 }
 
+void mission_start::asthmatic_npc( mission *miss )
```

Comment:
Very good idea! Done with "generalize lambda expression".
Now i think i should/must add a few others new missions (e.g. Add new mission: get insulin and others).